test/api/api_test.go

```diff
@@ -360,6 +360,7 @@ func RunRequests(
 
 	// Optionally check API server stdout/stderr
 	expectedStdoutPath := "expected-server-stdout"
+	fmt.Println(stderr.String())
```

Comment:
To je zapomenutý, nebo to tu necháváš naschvál? 🙂

Reply:
dik :man_facepalming: